### ISSUE TITLE
CMake Policy CMP0042

### ISSUE THREAD

**Issue by lqi** [oclint/oclint#226]
```
CMake Warning (dev):
   Policy CMP0042 is not set: MACOSX_RPATH is enabled by default. Run “cmake --help-policy CMP0042” for policy details.  Use the cmake_policy command to set the policy and suppress this warning.
```

**Comment by ryuichis** [comment]
Addressed in a5f84a6e546a954cc589cba59add7a66f1c316d1
